frontend/src/components/UserItem/index.tsx

```diff
@@ -28,7 +37,10 @@ const UserItem: React.FC<UserItemProps> = ({ user }) => {
           ))}
         </S.CoffeeIconImageBox>
       </S.Box>
-      <Checkbox />
+      <Checkbox
+        onChange={handleChange}
+        checked={user.attendanceStatus === 'tardy' ? false : true}
+      />
```

Comment:
로딩 시 optimisticChecked를 보여주는 로직에 대한 건 위와 같습니다.
삼항연산자보다 룩업테이블을 선언하면 가독성이 더 좋을 것 같아요.

```suggestion
      <Checkbox
        onChange={handleChange}
        checked={loading ? optimisticChecked : ATTENDANCE_STATUS[user.attendanceStatus]}
      />
```